query_base.go

```diff
@@ -57,17 +45,29 @@ type IDB interface {
 	NewTruncateTable() *TruncateTableQuery
 	NewAddColumn() *AddColumnQuery
 	NewDropColumn() *DropColumnQuery
+
+	HasFeature(feat feature.Feature) bool
+	Formatter() schema.Formatter
+	Table(reflect.Type) *schema.Table
+
+	_flags() internal.Flag
+	makeQueryBytes() []byte
+
+	afterQuery(
+		ctx context.Context,
+		res sql.Result,
+		err error,
+	)
```

Comment:
This was a big interface before and now it grew even bigger and got some private methods. Not nice :)